### PR TITLE
Update pt_BR.php

### PR DESCRIPTION
**Pull request by carlosstenzel**
Updating and repairs

### CODE REVIEW

File: app/var/languages/pt_BR.php [hunk 1]
```diff
@@ -213,8 +224,10 @@
         'thank_you'                       => 'Thank You!',
         'about_thank_you_1'               => 'We would like to thank all of our contributors and members of the community for pushing us to make Phalcon the best framework of its kind. Your kind words and engagement is what drives us to do better. We hope to meet and exceed the trust you put in Phalcon.',
         'flying_with_phalcon'             => 'Thanks for flying with Phalcon! &lt;3',
-
         'hosting_description'             => 'Phalcon can be installed on any machine on which the hosting provider offers SSH root access (or sudo). Below is a list of reputable companies that offer such services:',
+	    'hosting_5hosting_title'	      => '5hosting.com - unlimited Webspace',
+	    'hosting_5hosting_1'		      => '5hosting.com offers unlimited webspace, traffic, databases and email accounts in a high availability load balanced cluster based solely on SSD harddrives for a fixed monthly rate.',
+	    'hosting_5hosting_2'		      => "The phalcon framework comes pre-compiled for all stable PHP versions (5.4, 5.5 and 5.6) and can easily be activated with this <a href=':1:'>tutorial</a>.",
```
Comment (sergeyklay) [inline app/var/languages/pt_BR.php]: indents

File: app/var/languages/pt_BR.php [hunk 2]
```diff
@@ -34,7 +34,7 @@
         'profiling'                       => 'Profiling',
         'sharding'                        => 'Sharding',
         'odm_for_mongo'                   => 'ODM para Mongo',
-        'template_engine_volt'            => 'Engine de Template  (Volt)',
+        'template_engine_volt'            => 'Template Engine (Volt)',
```
Comment (sergeyklay) [inline app/var/languages/pt_BR.php]: rly? :)

Comment (carlosstenzel) [inline app/var/languages/pt_BR.php]: I did not see the error kk

**Comment by niden** [comment]
@carlosstenzel Thank you for all the help on this. Could you please use Crowdin to add your translations? Sorry for taking this long to respond to your PR.

https://crowdin.com/project/phalcon-website

**Pull request closed without merging**